core/src/test/scala/org/apache/spark/sql/delta/expressions/InterleaveBitsSuite.scala

```diff
@@ -98,4 +158,26 @@ class InterleaveBitsSuite extends SparkFunSuite with ExpressionEvalHelper {
     InterleaveBits(Seq(Literal(0.toDouble))).checkInputDataTypes() != TypeCheckSuccess
     InterleaveBits(Seq(Literal(0.toString))).checkInputDataTypes() != TypeCheckSuccess
   }
+
+  test("randomization interleave bits") {
+    val numIters = sys.env
+      .get("NUMBER_OF_ITERATIONS_TO_INTERLEAVE_BITS")
+      .map(_.toInt)
+      .getOrElse(1000000)
+    var i = 0
+    while (i < numIters) {
+      // generate n columns where 1 <= n <= 8
+      val numCols = Random.nextInt(8) + 1
+      val input = new Array[Int](numCols)
+      var j = 0
+      while (j < numCols) {
+        input(j) = Random.nextInt()
+        j += 1
+      }
+      val r1 = InterleaveBits.interleaveBits(input, true)
+      val r2 = InterleaveBits.interleaveBits(input, false)
+      assert(java.util.Arrays.equals(r1, r2), s"input: ${input.mkString(",")}")
```

Comment:
is this enough information to debug? how will in the input be printed with mkString? also.. should we print the output arrays as well to compare the bits with and without fast interleaving?

Reply:
> how will in the input be printed with mkString?

It would be something like
```
[info] InterleaveBitsSuite:
[info] - randomization interleave bits *** FAILED *** (32 milliseconds)
[info]   java.util.Arrays.equals(r1, r2) was false input: 1922396969,1778178185,1971203485 (InterleaveBitsSuite.scala:179)
[info]   org.scalatest.exceptions.TestFailedException:
[info]   at org.scalatest.Assertions.newAssertionFailedException(Assertions.scala:472)
[info]   at org.scalatest.Assertions.newAssertionFailedException$(Assertions.scala:471)
[info]   at org.scalatest.Assertions$.newAssertionFailedException(Assertions.scala:1231)
[info]   at org.scalatest.Assertions$AssertionsHelper.macroAssert(Assertions.scala:1295)
......
```

Reply:
> should we print the output arrays as well to compare the bits with and without fast interleaving?

Since the results are deterministic, we can easily reproduce them for debugging by running the code `InterleaveBits.interleaveBits(input, false)`